### PR TITLE
Try to fix memory problem

### PR DESCRIPTION
**Pull request by howard0su**

### CODE REVIEW

File: torchx/c_src/torchx.cpp [hunk 1]
```diff
@@ -129,15 +137,22 @@ create_tensor_resource(ErlNifEnv *env, torch::Tensor tensor)
   ret = enif_make_resource(env, tensorPtr);
   enif_release_resource(tensorPtr);
 
+  printf("Tensor %p is created\n", tensorPtr);
```
Comment (polvalente) [inline torchx/c_src/torchx.cpp:140]: I think this should be using std::cout if this is going to be kept in the code

```suggestion
  std::cout << "Tensor "<< tensorPtr << " is created" << std::endl;
```

Comment (howard0su) [reply torchx/c_src/torchx.cpp:140]: this is just for quick debug. I will remove it after I sort out the problem

Comment (polvalente) [reply torchx/c_src/torchx.cpp:140]: Don't mind me then haha!

**Comment by josevalim** [comment]
Hi @howard0su! Is this PR is still relevant? :)

**Comment by josevalim** [comment]
Continuing in #758

**Pull request closed without merging**